src/model/consent/consent.ts

```diff
@@ -83,20 +83,51 @@ export class ConsentDB {
   }
 
   // Update Consent
-  // No validation against Null or illegal updates in models
   public async update (consent: Consent): Promise<number> {
     // Returns number of updated rows
-    const updateCount: number = await this
-      .Db<Consent>('Consent')
-      .where({ id: consent.id })
-      .update(consent)
-
-    // Ensure that the caller knows that the resource does not exist
-    if (updateCount === 0) {
-      throw new NotFoundError('Consent', consent.id)
-    }
-
-    return updateCount
+    // Transaction to make the update atomic
+    return this.Db.transaction(async (trx): Promise<number> => {
```

Comment:
Generally, I think shorthand for this is `txn`.

Reply:
I was following these links which use `trx`:
 - http://knexjs.org/#Transactions
 - https://vincit.github.io/objection.js/guide/transactions.html